library/general/src/lib/y2issues/reporter.rb

```diff
@@ -37,45 +38,101 @@ class Reporter
     def initialize(issues, report_settings: Yast::Report.Export)
       textdomain "base"
       @presenter = Presenter.new(issues)
-      @level = issues.fatal? ? :error : :warn
+      @level = issues.error? ? :error : :warn
       @log, @show, @timeout = find_settings(report_settings, @level)
```

Comment:
@imobachgs I was thinking about this. We are deciding which AutoYaST settings to use (`<warnings>` vs `<errors>`) based on the severity. But I was wondering whether it would make sense to use `<yesno_messages>` in the cases in which we ask the user whether to continue or not (when `ask` is used for the corresponding severity).

Reply:
And talking about honoring the AutoYaST settings, what's the rationale to ignore the timeout and enforce it to zero in for the "error" level?